docs/samples/storage/azure/README.md

```diff
@@ -51,4 +54,28 @@ kubectl apply -f azcreds.yaml
 Note: To use your model binary you must reference the folder where it's located with an ending ```/``` to denote it`s a folder.
 ```bash
 https://accountname.blob.core.windows.net/container/models/iris/v1.1/
-```
\ No newline at end of file
+```
+
+### Use Pod Identity
+
+You may assign Managed Identity to `InferenceService` resource using Pod Identity.
+
+* Set up Pod Identity follow the steps [here](https://azure.github.io/aad-pod-identity/docs/demo/standard_walkthrough/)
+* Add the `aadpodidbinding` label to your service 
+
+```yaml
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kserve-simple-string
+  labels:
+    aadpodidbinding: piselector
+spec:
+  template:
+    metadata:
+  predictor:
+    serviceAccountName: sa
+    tensorflow:
+      storageUri: "https://kserve.blob.core.windows.net/triton/simple_string/"
```

Comment:
Does this url exist?

Reply:
It doesn't exist.
I don't think the previous one https://kfserving.blob.core.windows.net/triton/simple_string/ work as well but the kfserving.blob.core.windows.net exists.
Do we need to change the naming to kserve.blob.core.windows.net or stay in kfserving.blob.core.windows.net?